phones/iq_utils.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+import requests
+
+from django.conf import settings
+
+from rest_framework import exceptions
+
+
+def send_iq_sms(to_num: str, from_num: str, text: str) -> None:
```

Comment:
👍 Thanks for moving this. I agree, "utils" is often a code smell for me, but I think it is better to have these in the same place. I know some object-oriented ways to solve this, like a class `InteliquentService` that inherits from `BasePhoneService` and has a method `send_sms()`. We may want something like that to support both services. However, that feels like a future PR, since it would touch the Twilio stuff too, and this is a good focused PR.